components/vim/vim-modern.p5m

```diff
@@ -0,0 +1,27 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Denys Rtveliashvili
+#
+
+set name=pkg.fmri value=pkg:/editor/vim@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Global settings file to make VIM be actually IMproved by default"
+set name=pkg.description value="Vim is a clone of the Unix editor 'vi'.  By default it runs in an obsolete, compatible mode. These settings make it work in a non-compatible mode, which is assumed to be used in the VIM guide."
+set name=com.oracle.info.description value="Settings for the Vi IMproved text editor"
+set name=info.classification value="Development/Editors"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+file files/vimrc path=usr/share/vim/vimrc
+
+license vim.license license="VIM License"
+
+depend type=require fmri=editor/vim/vim@$(IPS_COMPONENT_VERSION)
```

Comment:
Depend on the base package `editor/vim@...` (not `vim/vim`) here. Maybe consider the whole suffix =>

````
depend type=require fmri=editor/vim@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION) 
````